Lesson22/selcan/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>To-Do App</title>
+		<link rel="stylesheet" type="text/css" href="./styles.css"/>
+		
+	</head>
+	<body>
+		<div class="container">
+			<h1>To-Do List</h1>
+			<!-- 
+				Form element with text input
+
+				ul > li > span & button
+
+				1. Add form element with text input, form and input should have id attributes
+				2. Add ul element with id that will be used in JS
+				3. Add example li with placeholder text to see how it looks on the screen
+				4. Select form element in JS, and get the input from the form when a form is submitted
+				5. In JS, create actual li element for the submitted todo
+			 -->
+			<form id="toDoForm">
+				<input type="text" placeholder="Enter to do" id="toDoInput"/>
+			</form>
+			<ul id="toDoList">
+				<!-- TODO: remove example li element when JS code is completed -->
+				<li>
```

Comment:
Remove this hardcoded <li> element so we only have todo elements added by a user